### PR TITLE
Add_layer

### PR DESCRIPTION
**Pull request by Jayas121**
added responsive and color over image

### CODE REVIEW

File: pages/branding.tsx [hunk 1]
```diff
@@ -5,15 +5,28 @@ import img2 from './branding.jpg'
 export default function mobile() {
   return (
   <>
-  <header className="h-80 w-100  bg-blue text-center">
+  <header className="h-96 w-100  bg-bluer text-center bg-blue bg-blend-multiply" style={{
```
Comment (gitgenjutsu) [inline pages/branding.tsx]: bg-bluer - remove this

File: pages/mobile.tsx [hunk 2]
```diff
@@ -20,12 +20,16 @@ export default function mobile() {
...
       <div className="pl-10 text-xl">
         
-        <header className="h-60 w-100 bg-blue text-white px-20 py-10 text-center">   
-        <div className="flex  justify-center flex-col">
-          <h1 className="text-5xl font-semibold">MOBILE APPLICATION DEVELOPMENT
-          </h1>
-          </div>
-          </header>
+        
+          <header className="h-96 w-100  bg-bluer text-center bg-blue bg-blend-multiply" style={{
```
Comment (gitgenjutsu) [inline pages/mobile.tsx:25]: remove - bg-bluer

File: tailwind.config.js [hunk 3]
```diff
@@ -15,8 +19,21 @@ module.exports = {
       
       
         },
+
+        screens:{
+          sm: "370px",
+          md: "990px",
+          lg: "1200px",
+          xl: "1280px",
+          "2xl": "1536px",
```
Comment (gitgenjutsu) [inline tailwind.config.js]: "2xl" - should not be in strings

**Review by gitgenjutsu — CHANGES_REQUESTED** [review]
Solve comments

**Comment by Jayas121** [comment]
corrected my mistake

**Pull request merged**